src/main/java/az/ingress/Application.java

```diff
@@ -0,0 +1,17 @@
+package az.ingress;
+
+import az.ingress.configuration.QueueProperties;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+
+import static org.springframework.boot.SpringApplication.run;
+
+@EnableConfigurationProperties(QueueProperties.class)
```

Comment:
you can use `@ConfigurationPropertiesScan` instead of `@EnableConfigurationProperties`
( when a new config is added, there will be no need to change it )